zproject/prod_settings_template.py

```diff
@@ -764,6 +764,22 @@
 ## Controls whether Zulip will rate-limit user requests.
 # RATE_LIMITING = True
 
+## Entries in this dictionary will override Zulip's default rate
+## limits. Rules for domains which are not explicitly overridden here
+## will be as default. View the current rules and options using:
+##   /home/zulip/deployments/current/scripts/get-django-setting RATE_LIMITING_RULES
+##
+## The limits are tuples of a number of seconds and a number of
+## requests allowed over that many seconds. For example, to change the
```

Comment:
Maybe also explicitly say something like:
> If multiple tuples are given, breaching _any_ of them will trigger a rate-limited response to the client.

Reply:
@mateuszmandera, I moved where you inserted my suggested sentence to be after this first sentence, where I feel it makes a little more sense.